tests/testsuite.src/syn_literals.at

```diff
@@ -1511,3 +1511,27 @@ prog.cob:5: error: missing terminating ' character
 ])
 
 AT_CLEANUP
+
+
+AT_SETUP([GCOS literals with EBCDIC symbols (syntax)])
+AT_KEYWORDS([extensions gcos ebcdic])
+AT_DATA([prog.cob],[IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       PROCEDURE        DIVISION.
+           DISPLAY ""135,151,151"bar"195,194,0""
+           DISPLAY ""135,257""
+           DISPLAY ""135,x,194,1""
+           DISPLAY ""135,,155,""
+           DISPLAY ""135  155 " "
+
+           STOP RUN.
+])
+AT_CHECK([$COMPILE -febcdic-symbolic-characters prog.cob], [1], [], [prog.cob:1: note: free format detected
+prog.cob:4: error: symbolic character 0 out of allowed EBCDIC range (1...256)
+prog.cob:5: error: symbolic character 257 out of allowed EBCDIC range (1...256)
+prog.cob:6: error: invalid character 'x' in sequence of symbolic EBCDIC characters
+prog.cob:7: error: extraneous character ',' in sequence of symbolic EBCDIC characters
+prog.cob:7: error: extraneous separator ',' at end of symbolic EBCDIC characters
+])
```

Comment:
please add a program with valid values (similar to the one in run_extensions), then compile it with and without `-febcdic-symbolic-characters` and one time with `-febcdic-symbolic-character -febcdic-table=dummyNotThere`.